### ISSUE TITLE
(enhancement) different way to specify auto-remove minimum space

### ISSUE THREAD

**Issue by Germar** [bit-team/backintime#350]
Currently, one must specify the auto-remove minimum space in terms of a fixed size value.  However, I believe it would be more beneficial if it were possible to specify the minimum space in reference to how much each individual backup takes up.  For instance, something like 10, 25, or 50% more than last backup.

**Comment by emtiu** [comment]
I'm not sure this is possible in the way that the reporter envisioned it. It would at least require an implementation of #406. Or did I misunderstand?

In any case, relative values for free space would be nice to have.

**Comment by buhtz** [comment]
Might not be directly relevant to this issue. But I investigated the current auto-/smart-remove behavior and documented it. I found inconsistent behavior and minor bugs. There is a lot potential to improve and refactoring.

Before starting with this I would like to request your opinions and suggestions about the next steps. Please see the issue #1945 for details and a mockup for the new auto-remove dialog-tab.

Feedback and suggestions welcome. We appreciate your feedback.

**Comment by buhtz** [comment]
I tend to close the issue without a solution. It is a nice to have thing. But on the other side this tiny feature need a high amount of work. Cost-benefit-relation is not good, I would say.

**Comment by buhtz** [comment]
Closing this ticket based on the comment above. Feel free to reopen
if the problem still exists. Thank you for your efforts. If you have
any further questions, ideas or encounter any other issues, please
don't hesitate to let us know.

Best regards,
Christian
